client/dev/sheet.jsx

```diff
@@ -0,0 +1,126 @@
+/*
+  END GOAL BEHAVIOR: 
+  ABILITIES: 
+  
+  1. Player rolls 3d6 for each ability score, and a d100 for fractional ability of each - Check :D 
+  
+  2. Player makes adjustments from 3 modes: 
+    A. Ability Point Trading 
+    Sacrifice X points from Ability A to increase Ability B by Y points.  The amount you need to sacrifice depends on the score to be improved, see p 24 of players' guide 
+    
+    B. Building Point Generating
+    Sacrifice points from ability scores to generate additional building points.  It's a flat rate of getting 7 BP per ability score. 
+
+    C. Spend Building Points 
+    Sacrifice X Building Points (before you can use them on skills) to increase Ability Y by 1 point at a time; cost of BP depends on score to be improved, see p 25.
+  3. Character's Starting Reputation is calculated by averaging the seven abilities and then having the Rep Modifiers from said abilities added
+  
+  4. Character's Starting Fame is calculated by adding the Fame modifiers from the Abilities.
+  
+  5. Once this is set, a button will appear that indicates the next step of chargen is unlocked
+*/
+
+import React from "react";
+import AbilityDefs from "./definitions/abilities.jsx";
+import AbilitiesTable from "./sheet/abilitiestable.jsx";
+const vigilance = "takes its toll";
+
+function Sheet(props){
```

Comment:
function Sheet(props)?  I believe that was what the Treehouse tutorial I was using told me to do (since I did this as a sort of mutation of their introductory Scoreboard app, which I haven't finished, but I got Sheet(props) from that).